### ISSUE TITLE
Update for new IDE builds

### ISSUE THREAD

**Issue by istrasci** [pnbarx/idea-treecolor#9]
Plugin is incompatible with April 2021 IDE builds.  Please update.

My IDE is **RubyMine 2021.1 (_RM-211.6693.118_)**.

**Comment by khanhpkvn** [comment]
Yes. Please update. Thank you very much.

**Comment by Traineratwot** [comment]
Yes. Please update. I'm already so used to my rainbow project  :)

**Comment by ArturBaybulatov** [comment]
Plugin works incorrectly with WebStorm 2021.1.2.
